VogtPayroll2/Program.cs

```diff
@@ -1,12 +1,73 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace VogtPayroll2
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            PayrollManager payrollManager = new PayrollManager();
+            List<Employee> employeeList = new List<Employee>();
+
+            employeeList = PrintMenu();
+
+            foreach (var employee in employeeList)
+            {
+                employee.NetPay = employee.GetNetPay();
+                employee.DisplayEmployeeInfo(employee);
+
+            }
+
+            employeeList = payrollManager.GetNetPayOfAllEmployees(employeeList);
```

Comment:
You're not actually doing anything with these functions. You get the results, but I dont see you pritning them.

If you want to print them, you should make the GetNetpayOfAllEmployee/GetMinimumAndMaximumNetPay private functions and instead, expose a public PrintNetPayOfAllEmployees/etc. If thats the purpose of those functions is to print the employees net pay sorted.